MediaGalleryUi/view/adminhtml/web/js/grid/massaction/massactionView.js

```diff
@@ -104,11 +106,12 @@ define([
 
             $(this.deleteButtonSelector)
                 .removeClass('media-gallery-actions-buttons')
-                .text($.mage.__('Delete Selected'))
+                .text($t('Delete Selected'))
                 .addClass('primary');
 
             if (!$(this.cancelMassactionButtonSelector).length) {
                 $(this.pageActionsSelector).append(this.cancelMassactionButton);
+                $('#cancel_massaction').text($t('Cancel'));
```

Comment:
Can we use the `translate="Cancel"` attribute in the template instead?

Reply:
If that does not work by default, it might be needed to add a default `cancelMassactionSelector: '#cancel_massaction'` and `$(this.cancelMassactionSelector).applyBindings();`